.github/DISCUSSION_TEMPLATE/gateway.yml

```diff
@@ -60,96 +54,61 @@ body:
+      description: "Please include a short summary (max 5 dot points) about your chain's technology."
     validations:
       required: true
 
-  - type: markdown 
-    attributes:
-      value: |
-       "# Native token"
   - type: textarea 
     id: token 
     attributes:
       label: Native token details 
-      description: "Details about the native token and tokenomics"
+      description: "Details about the native token and tokenomics."
       value: |
+        <!---
+        Please include at least:
         - The max supply of tokens.
         - Use cases for the token (i.e. governance, fees, etc).
+        -->
     validations:
       required: true
 
-
-  - type: markdown 
-    attributes:
-      value: |
-       "# Node operators"
   - type: textarea 
     id:  node-operators 
     attributes:
       label: Technical requirements for running a node 
-      description: "Any extra requirements for Guardians"
-      value: |
-        If there are no extra requirements for guardians, please use the message below: 
-        None - [chain] will emit messages to Gateway via IBC. Guardians should not have to run anything additional (except for guardians who may run IBC relayers). 
+      description: "If there are no extra requirements for guardians, please use the following message: `None - [chain] will emit messages to Gateway via IBC. Guardians should not have to run anything additional (except for guardians who may run IBC relayers).`"
     validations:
       required: true
 
-
-  - type: markdown 
-    attributes:
-      value: |
-       "# IBC Relaying Strategy"
   - type: textarea 
     id: ibc-relaying 
     attributes:
       label: IBC Relaying Strategy 
-      description: "Any extra requirements for Guardians"
+      description: "Please describe how IBC relaying between Wormhole Gateway and your chain will be handled."
       value: |
-        Please describe how IBC relaying between Wormhole Gateway and your chain will be handled.
+        <!---
         - Are you asking one of the Wormhole Guardians to handle IBC relaying?
         - Do you already have an IBC relaying provider who is not a Wormhole Guardian? If so, please include as much information as possible on who they are and what kind of service agreement you have with them.
+        -->
     validations:
       required: true
-
-  - type: markdown 
-    attributes:
-      value: |
-       "# RPC"
-  - type: textarea 
-    id: rpc 
-    attributes:
-      label: RPC 
-      description: "Describe which dedicated RPCs are for both mainnet and testnet are available"
-      value: |
-        Wormhole marquee UIs such as [Portal Bridge](https://portalbridge.com/) and [Connect](https://wormhole.com/connect/) require dedicated RPCs to provide the best user experience. We ask that you coordinate with an RPC provider to set these up for the Wormhole UIs to use.
-    validations:
-      required: true
```

Comment:
no need for this anymore?

Reply:
seems like they should be available at https://github.com/cosmos/chain-registry